internal/build/job/maker.go

```diff
@@ -18,13 +14,37 @@ import (
 	"k8s.io/utils/pointer"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+
+	kmmv1beta1 "github.com/kubernetes-sigs/kernel-module-management/api/v1beta1"
+	"github.com/kubernetes-sigs/kernel-module-management/internal/build"
+	"github.com/kubernetes-sigs/kernel-module-management/internal/constants"
+	"github.com/kubernetes-sigs/kernel-module-management/internal/utils"
+)
+
+const (
+	dockerfileVolumeName = "dockerfile"
 )
 
 //go:generate mockgen -source=maker.go -package=job -destination=mock_maker.go
 
+type JobTemplateOptions struct {
+	ModuleSpec kmmv1beta1.ModuleSpec
+
+	BuildConfig *kmmv1beta1.Build
+
+	TargetKernel string
+
+	ContainerImage string
+
+	ShouldPushImage bool
+
+	RegistryTLS *kmmv1beta1.TLSOptions
+
+	Owner metav1.Object
```

Comment:
Could we remove the empty lines? Would make everything nice and aligned 🙂

Reply:
This change has been reverted, so only for the sake of argument, the empty lines are pretty useful IMHO. At first they might seem a bit out of order, but they help a lot regarding version control and the future diffs, for example when one of those lines is removed or another one is added. 

For this specific change, I should have also added the doc strings for every one of those exported fields. This way everything would be clearer, more maintainable and easier to diff in the future.